core/textile/bucket/bucket_dir.go

```diff
@@ -80,3 +80,31 @@ func (b *Bucket) DeleteDirOrFile(ctx context.Context, path string) (path.Resolve
 
 	return b.bucketsClient.RemovePath(ctx, b.Key(), path)
 }
+
+// return the recursive items count for a path
+func (b *Bucket) ItemsCount(ctx context.Context, path string) (int32, error) {
+	b.lock.RLock()
+	defer b.lock.RUnlock()
+
+	var count int32
+
+	dir, err := b.ListDirectory(ctx, path)
+	if err != nil {
+		return 0, err
+	}
+
+	count = dir.Item.ItemsCount
+
+	for _, item := range dir.Item.Items {
```

Comment:
Maybe we want to skip the meta files such as `.textileseed`

Reply:
@dmerrill6 

Thank you! Using `utils.IsMetaFileName` now in cb27edfd578d8703e4ab7880e61e3326f7a70ad2.